fe1-web/src/core/network/validation/schemas/dataSchemas.ts

```diff
@@ -1,22 +1,22 @@
+import dataAddChirp from 'protocol/query/method/message/data/dataAddChirp.json';
```

Comment:
Should we disable the rule in this file ? 
It's a lot harder to find something or identify a missing import now 😅

Reply:
That seems to be a good idea yeah... 😅